logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/serializer/DataFrameDescriptor.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.logstreams.impl.serializer;
+
+import static org.agrona.BitUtil.align;
+
+import org.agrona.MutableDirectBuffer;
+
+/**
+ * Remnants of the dispatcher data framing. A 12 byte header starting with the frame length. The
+ * rest of the 12 bytes are unused. This is kept for backwards-compatability reasons because {@link
```

Comment:
```suggestion
 * rest of the 12 bytes are unused. This is kept for backwards-compatibility reasons because {@link
```

Reply:
Damn, that's one of the word I never get right on the first try :D